clickhouse/tests/test_clickhouse.py

```diff
@@ -34,6 +35,8 @@ def test_check(aggregator, instance, dd_run_check):
     )
 
     aggregator.assert_service_check("clickhouse.can_connect", count=1)
+    aggregator.assert_all_metrics_covered()
```

Comment:
What about comment on line 18?

Reply:
The attempt here is to add all the optional metrics to the OPTIONAL_METRICS list. But yes, this PR might introduce some flakeynes that should be easy to address (by adding any missing metric)